pkg/test/extensions/binary.go

```diff
@@ -217,6 +217,10 @@ var extensionBinaries = []TestBinary{
 		imageTag:   "cluster-kube-storage-version-migrator-operator",
 		binaryPath: "/usr/bin/cluster-kube-storage-version-migrator-operator-tests-ext.gz",
 	},
+	{
+		imageTag:   "cluster-ingress-operator",
+		binaryPath: "/usr/bin/ingress-operator-ext-tests.gz",
```

Comment:
To be consistent with other team, could we rename the binary file as `cluster-ingress-operator-tests-ext.gz` ?